tests/modules/nf-core/kmcp/index/nextflow.config

```diff
@@ -0,0 +1,10 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: KMCP_INDEX {
+        ext.prefix = { "${meta.id}_" }
+        ext.args = { "--out-dir ${task.ext.prefix}" }
```

Comment:
This is a duplciate of the code it self right?
```suggestion
```

Reply:
I added it because of the error:
```
Command error:                                                                                                                                            
           11:53:13.202 [ERRO] intput and output paths should not be the same 
```    
It is the same as here https://github.com/nf-core/modules/blob/master/tests/modules/nf-core/kmcp/compute/nextflow.config

Reply:
> I added it because of the error:
> 
> ```
> Command error:                                                                                                                                            
>            11:53:13.202 [ERRO] intput and output paths should not be the same 
> ```
> 
> It is the same as in `kmcp/compute` https://github.com/nf-core/modules/blob/master/tests/modules/nf-core/kmcp/compute/nextflow.config

That should be solved with changing the `prefix` though, no?

Reply:
Would it be a better practice to change?

`ext.args = { "--out-dir ${task.ext.prefix}" }`

Reply:
No - I woudl drop iot entirely. As `--out-dir` is influencing an output channel, this should be controlled entirely within the module itself (ratehr than args).

Reply:
Then I should update the `kmcp/compute` as well.

Reply:
OK! Yes, not good to duplciate a parameter regardless, pick one or another :)

Reply:
Great, thanks :)